detection_rules/rule_validators.py

```diff
@@ -371,7 +372,15 @@ def validate(self, data: "QueryRuleData", meta: RuleMeta, max_attempts: int = 10
                         # auto add the field and re-validate
                         self.auto_add_field(validation_checks["stack"], data.index_or_dataview[0])
                     else:
-                        raise ValueError(f"Error in both stack and integrations checks: {validation_checks}")
+                        stack_error_msg = validation_checks["stack"].error_msg
+                        integration_error_msg = validation_checks["integrations"].error_msg
+                        stack_error_trace = validation_checks["stack"]
+                        integration_trace = validation_checks["integrations"]
+                        click.echo(f"Stack Error Message: {stack_error_msg}")
```

Comment:
Nit, you may want to do something like this to make it more obvious that it is an error in the click output. 
```suggestion
                        click.secho(f"ERROR: Stack Error Message: {stack_error_msg}", fg="red")
                        click.secho(f"ERROR: Stack Error Trace: {stack_error_trace}", fg="red")
                        click.secho(f"ERROR: Integrations Error Message: {integration_error_msg}", fg="red")
                        click.secho(f"ERROR: Integrations Error Trace: {integration_trace}", fg="red")
```

Testing code and sample output

<details><summary>Code</summary>
<p>

```python
import click
stack_error_msg = "Stack Error Message"
click.secho(f"ERROR: Stack Error Message: {stack_error_msg}", fg="red")
raise ValueError("Error in both stack and integrations checks")
```

</p>
</details> 

Output:
![image](https://github.com/user-attachments/assets/0c4473ea-2d30-45bd-8cee-614f1f0ad20d)

Reply:
But when we run in via console I am not seeing any difference :) 

more like `python -m detection_rules test` which will be invoked in the github pipeline as well 

<img width="1192" alt="image" src="https://github.com/user-attachments/assets/203b9af3-8cbe-4dad-bb3e-d9252190533b" />

Reply:
Ah because its captured, good call :+1:

Reply:
Resolving as this does not have any effect on the pipeline run